vendor/css/custom_bootstrap.css

```diff
@@ -0,0 +1,103 @@
+/* This is custom bootstrap */
+
+.container{
+    width: 90%;    
+}
+
+.container-80{
+    width: 80%;    
+}
+
+.container-85{
+    width: 85%;    
+}
+
+.m-auto{
+    margin: auto;
+}
+
+.w-100{
+    width:100%;
+}
+
+.flexbox{
+    display: flex;
+}
+
+.shadow{
+    box-shadow: var(--Shadow) 3px 5px 25px !important;
+}
+
+.shadow-1{
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 0px 8px !important;
+}
```

Comment:
it is not right way to make multiple shadow versions you can use
card-shard
links-shadow 
flag-shadow